private/sync2/tee.go

```diff
@@ -31,6 +33,7 @@ func NewTeeFile(readers int, tempdir string) ([]PipeReader, PipeWriter, error) {
 	if err != nil {
 		return nil, nil, err
 	}
+	os.Remove(filepath.Join(tempdir, tempfile.Name()))
```

Comment:
On Windows this is getting when checking the error:
```
$ go test ./private/sync2
--- FAIL: TestTee_CloseWithError (0.00s)
    --- FAIL: TestTee_CloseWithError/File (0.00s)
        tee_test.go:104: remove Z:\Temp\tee935932101: The process cannot access the file because it is being used by another process.
--- FAIL: TestTee_Basic_On_Disk (0.00s)
    --- FAIL: TestTee_Basic_On_Disk/File (0.00s)
        tee_test.go:104: remove Z:\Temp\tee648300414: The process cannot access the file because it is being used by another process.
FAIL
```

Reply:
Neat... I'll investigate some more.